app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -324,7 +325,11 @@ const ListContainer = (props: {
             />
           </>
         ) : (
-          <Text>Loading...</Text>
+          <>
```

Comment:
Please could you also remove the `Text` import at the top of the file which isn't needed now

Reply:
Done!